DOCUMENTATION.md

```diff
@@ -118,5 +118,5 @@ https://github.com/gohugoio/hugo/releases
 2. Generate the website using the updated version: `hugo -d /tmp/updated`
 3. Check the diff to be sure nothing is broken: `diff -r /tmp/current /tmp/updated` (the diff is **never** empty, you must see at least the version updated in two files per languages)
 4. If necessary, updates the code to be compatible with the new Hugo version
-5. Update `netlify.toml` and `.travis.yml` to the new Hugo version
+5. Update `netlify.toml` to the new Hugo version
```

Comment:
Wouldn't you want to indicate "test.yaml" here as well, as you are updating it in this PR?

Reply:
@deining Happy to merge once this is addressed

Reply:
> Wouldn't you want to indicate "test.yaml" here as well, as you are updating it in this PR?

Yes, good idea. The PR is changed accordingly now.